client/state/data-layer/wpcom/jetpack-checkout/support-ticket/index.js

```diff
@@ -31,11 +31,13 @@ const onUpdateSuccess = ( action, response ) => {
 		{
 			type: JETPACK_CHECKOUT_UPDATE_SUPPORT_TICKET_REQUEST_SUCCESS,
 			receiptId: action.receiptId,
+			jetpackTemporarySiteId: action.jetpackTemporarySiteId,
 		},
 		{
 			type: JETPACK_CHECKOUT_UPDATE_SUPPORT_TICKET_RECEIVE,
 			receiptId: action.receiptId,
 			payload: response,
+			jetpackTemporarySiteId: action.jetpackTemporarySiteId,
```

Comment:
Is not guaranteed that the receipt ID will be available post-purchase so it's better to use the site ID of the temporary site.